packages/cli/src/commands/index.ts

```diff
@@ -104,6 +109,7 @@ export function registerScriptCommand(program: Command) {
     .command('start')
     .description('Start a package for local development')
     .option(...configOption)
+    .option(...disableWatchFileOption)
```

Comment:
Let's not add this here imo, it's a little bit pointless really. The Webpack DevServer will have a large number of watchers anyway. This also is only meant to be run in local development.